boards/seeed/wio_wm1110_dev_board/Kconfig.wio_wm1110_dev_board

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2025 Ishraq Ibne Ashraf <ishraq.i.ashraf@gmail.com>
+#
```

Comment:
can you change the commit message to only have one part, adding the board, and one sign off?